config.go

```diff
@@ -44,10 +44,20 @@ var (
 	effectiveConfigGOPATHs []string
 )
 
+const srclibGopath = ".srclib-gopath"
```

Comment:
Yes, that's in the repo/build directory, just like `.srclib-cache` and `.srclib-store`.